gc.h

```diff
@@ -6,10 +6,12 @@
 #define SET_MACHINE_STACK_END(p) __asm__ __volatile__ ("movq\t%%rsp, %0" : "=r" (*(p)))
 #elif defined(__i386) && defined(__GNUC__)
 #define SET_MACHINE_STACK_END(p) __asm__ __volatile__ ("movl\t%%esp, %0" : "=r" (*(p)))
-#elif (defined(__powerpc__) || defined(__powerpc64__)) && defined(__GNUC__) && !defined(_AIX)
+#elif (defined(__powerpc__) || defined(__powerpc64__)) && defined(__GNUC__) && !defined(_AIX) && !defined(__APPLE__) // Not Apple is needed to unbreak ppc64 build on Darwin
```

Comment:
If `__powerpc__` is not defined on macOS, is this needed?

Reply:
@nobu Yes, I know, and ppc (ppc32) builds fine without this. However ppc64 fails – because this ELF code gets invoked. Don’t ask me why – I have no idea. It should not. But it does. It took me quite some time to figure out why ppc64 build kept failing with errors which are usually assembler-related. Here it is.